### PR TITLE
fix some bugs in project, add ui for search # and add test for AboutUs

### PR DESCRIPTION
**Pull request by Volodia1999**

### CODE REVIEW

File: client/src/components/Menu/index.tsx [hunk 1]
```diff
@@ -20,6 +20,12 @@ import {
     addNextResults,
 } from '../../store/search/actions';
 
+const search_hashtags = [
```
Comment (anna-kuzii) [inline client/src/components/Menu/index.tsx]: I believe it will change soon. Then TODO needs?

File: client/src/components/Menu/index.tsx [hunk 2]
```diff
@@ -149,6 +159,23 @@ class Menu extends React.Component<IMenuProps> {
                                                 </DropdownItem>
                                                 <DropdownItem divider/>
                                             </div>
+                                            {sharp === searchValue && search_hashtags.map((hashtag: any) => (
```
Comment (anna-kuzii) [inline client/src/components/Menu/index.tsx]: any?

File: client/src/components/Menu/index.tsx [hunk 3]
```diff
@@ -107,6 +113,8 @@ class Menu extends React.Component<IMenuProps> {
     public render(): JSX.Element {
         const {loggedUsername, newUsername, searchResults, loaded}: IMenuProps = this.props;
         const {searchValue}: IMenuState = this.state;
+        const sharp = 'e';
```
Comment (anna-kuzii) [inline client/src/components/Menu/index.tsx]: why e?

File: client/src/components/Menu/index.tsx (hunk 2 above)
Comment (anna-kuzii) [inline client/src/components/Menu/index.tsx]: do u need this const? search === 'e' isn't better?

File: client/src/components/Post/style.scss [hunk 4]
```diff
@@ -19,7 +19,7 @@
   }
 
   .comments {
-    padding-bottom: 30%;
+    min-height: 150px;
```
Comment (anna-kuzii) [inline client/src/components/Post/style.scss]: nice

File: client/src/components/Menu/index.tsx (hunk 3 above)
Comment (Volodia1999) [reply client/src/components/Menu/index.tsx]: I tested with this variable e, i changed on #

File: client/src/components/Menu/index.tsx (hunk 2 above)
Comment (Volodia1999) [reply client/src/components/Menu/index.tsx]: Fixed

Comment (Volodia1999) [reply client/src/components/Menu/index.tsx]: Fixed

File: client/src/components/Menu/index.tsx (hunk 1 above)
Comment (Volodia1999) [reply client/src/components/Menu/index.tsx]: Yes, it will change. I added TODO

**Pull request closed without merging**